docs/migrating-content-markdown-to-asciidoctor.adoc

```diff
@@ -0,0 +1,96 @@
+= Migration tips: from markdown to asciidoctor
```

Comment:
Do we intend to keep this documentation page forever or is it just for us for now?

Reply:
For sure, we will need it prior the actual migration: this should avoid looking at several sources (including our private Confluence) every time we want to migrate a doc content branch.

For a later usage, I don't know, it may be used as material for articles after the actual migration. It could be also helpful for people that want to do migration as we do now.